src/views/topics/TopicFormView.vue

```diff
@@ -223,6 +223,8 @@ onMounted(() => {
         setMetaTitle()
       })
       .finally(() => loader.hide())
+  } else {
```

Comment:
Since you're calling it in both paths, can you just move it after the top-level `if`?
```
  if (!props.isCreate || routeQuery.clone != null) {
    ...
  }
  setMetaTitle()
```

Reply:
Nope, cause the first branch is async and we want to apply at the end of the flow.